src/Core/src/Platform/Windows/ImageViewExtensions.cs

```diff
@@ -11,9 +12,13 @@ public static void Clear(this WImage imageView)
 			imageView.Source = null;
 		}
 
-		public static void UpdateAspect(this WImage imageView, IImage image)
+		public static void UpdateAspect(this WImage imageView, IImage image, FrameworkElement element)
```

Comment:
Wouldn't this cause a binary break? 
Also, a perfect time to add some documentation on what this method does ;)

Reply:
What do you mean by a binary break?

Reply:
By changing an already existing API, previous versions that used it the API can no longer use it unless they're updated to the most recent version of .net. 

I'm not 100% certain on the logistics of this, but I know @mandel-macaque  or @PureWeen  will know for sure :)

Reply:
@PureWeen Do you suspect this binary break will be an issue?

Reply:
Removed.